### PR TITLE
DO NOT MERGE - Release/2.1.0

### PR DESCRIPTION
**Pull request by stefaniacardenas**
* Updated CHANGELOG.md
* Updated JSFiddle link in README.md
* `BASE_36_VERSION` is now `BASE_32_VERSION`
* Updated release guidlines

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-277-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-277-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://277-pr-onfido-sdk-ui-onfido.surge.sh

File: webpack.config.babel.js [hunk 1]
```diff
@@ -135,10 +135,10 @@ const basePlugins = [
     'DESKTOP_SYNC_URL': CONFIG.DESKTOP_SYNC_URL,
     'MOBILE_URL' : CONFIG.MOBILE_URL,
     'SMS_DELIVERY_URL' : CONFIG.SMS_DELIVERY_URL,
-    // Increment BASE_36_VERSION with each release following Base32 notation, i.e AA -> AB
+    // Increment BASE_32_VERSION with each release following Base32 notation, i.e AA -> AB
     // Do it only when we introduce a breaking change between SDK and cross device client
     // ref: https://en.wikipedia.org/wiki/Base32
-    'BASE_36_VERSION' : 'AA',
+    'BASE_32_VERSION' : 'AB',
```
Comment (hjheath) [inline webpack.config.babel.js]: Do we need to bump this version? Has there been a breaking change?

Comment (stefaniacardenas) [reply webpack.config.babel.js]: The SMS feature is not a breaking change but it's a major change of the cross device feature so I thought we might want to keep track of that.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-277-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-277-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://277-pr-onfido-sdk-ui-onfido.surge.sh

File: webpack.config.babel.js (hunk 1 above)
Comment (dawiddylowicz) [reply webpack.config.babel.js]: According to guidelines, we should `Increment BASE_32_VERSION with each release...`. @hjheath do you challenge that? I think bumping version each time may mitigate the risk of forgetting to deploy when real changes occur, especially when many different people are starting to do the releases

**Comment by stefaniacardenas** [comment]
FYI I've fixed the style issues on codeclimate. The duplicate code issues and the CSS string detections are not accurate and should be disabled. I've opened a [Jira ticket](https://jira.onfido.co.uk/browse/CX-1558) for that.

File: webpack.config.babel.js (hunk 1 above)
Comment (hjheath) [reply webpack.config.babel.js]: yeah I think it makes sense to bump it for this release. for patch releases we should not need to.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-277-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-277-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://277-pr-onfido-sdk-ui-onfido.surge.sh

File: webpack.config.babel.js (hunk 1 above)
Comment (rfreitas) [reply webpack.config.babel.js]: so this is an interesting one. if there are no breaking changes, we can not bump it, but we will always redeploy it right?

And coming back to the version bumping, a UI refresh might not have a major bump, but if we don't distinguish the two, either the new version doesn't get the UI refresh or the old one gets it and it looks inconsistent.

**Pull request closed without merging**